status/authorization.py

```diff
@@ -19,14 +19,20 @@ def get(self):
                 "authorized/users", reduce=False
             )
             user_roles = self.application.gs_users_db.view(
-                "authorized/roles", reduce=False
+                "authorized/info", reduce=False
```

Comment:
I guess this is from this PR? https://github.com/NationalGenomicsInfrastructure/genomics-status/pull/915

Reply:
Yes, correct. This PR builds on top of that one. Weird that the commits are still here even after that one is merged?

Reply:
I guess you could pull in upstream/master?